### PR TITLE
Install on Kubernetes cluster docs no longer point to nonexistent manifests

### LINKED ISSUE #3453: Install on a Kubernetes Cluster section uses nonexistent yaml files
Option 1 and Option 2 on the [Install on a Kubernetes Cluster page](https://preliminary.istio.io/docs/setup/kubernetes/install/kubernetes/#installation-steps) instruct the user to `kubectl apply -f install/kubernetes/istio-demo.yaml` and `install/kubernetes/istio-demo-auth.yaml` respectively, when neither of those config files are included in the 1.1 release. I believe that the instructions need to be updated to use `helm template` to create similar manifests for the user to apply.

### PR DESCRIPTION
**Pull request by m-eaton**
Fixes #3453

### CODE REVIEW

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 1]
```diff
@@ -67,7 +67,12 @@ To install Istio with mutual TLS enabled and set to use permissive mode
 between sidecars:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]:
```suggestion
between sidecars run the following commands:
```

File: content/docs/setup/kubernetes/install/kubernetes/index.md [hunk 2]
```diff
@@ -85,7 +90,12 @@ To Install Istio and enforce [mutual TLS authentication](/docs/concepts/security
 between sidecars by default:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/install/kubernetes/index.md]:
```suggestion
between sidecars by default run the following commands:
```

**Comment by frankbu** [comment]
See comment in #3453

**Pull request closed without merging**